core/client/app/components/gh-posts-list-item.js

```diff
@@ -22,7 +22,7 @@ export default Component.extend(ActiveLinkWrapper, {
     }),
 
     authorAvatar: computed('post.author.image', function () {
-        return this.get('post.author.image') || this.get('ghostPaths.url').asset('/shared/img/user-image.png');
+        return this.get('post.author.image') || `${this.get('ghostPaths.subdir')}/ghost/img/user-image.png`;
```

Comment:
Are you aware of any themes that rely on those images? Ghost makes no attempt to make those images available to themes through helpers or any other method, it would normally be up to the theme developer to provide default images if they want them. In Casper for instance no default images are output because they are wrapped in checks such as `{{#if author.image}}<img ...>{{/if}}`.